apps/api/src/app/controllers/__tests__/__snapshots__/queryACLs.e2e.spec.ts.snap

```diff
@@ -0,0 +1,7081 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Access Control List and Role Based filtering in resource queries when querying for BibliographicReference when the user is a COSCRAD admin when querying for many resources (fetch many) should find all resources of the given kind 1`] = `
```

Comment:
I wonder if we really need to snapshot all of this? I suppose we do want to make sure the appropriate results (published, private with user in ACL, private to user) come back in each case... Maybe we could do it programmatically instead?

Reply:
We're already checking that the correct IDs are there. I think this is sufficient. The snapshot doesn't actually do any more than that, except to verify the view-model mapping, which is done in a different test suite.

Reply:
snapshot removed!